packages/material-ui/src/Popper/Popper.d.ts

```diff
@@ -19,7 +19,7 @@ export type PopperPlacementType =
 
 export interface PopperProps extends React.HTMLAttributes<HTMLDivElement> {
   transition?: boolean;
-  anchorEl?: null | Element | ReferenceObject | (() => Element);
+  anchorEl?: null | ReferenceObject | (() => ReferenceObject);
```

Comment:
My only concern here is that the code editors will suggest RefrenceObject, something that is more obscure than Element.

Reply:
We can alias this to `AnchorElement`.

Reply:
Like this?
```diff
diff --git a/packages/material-ui/src/Popper/Popper.d.ts b/packages/material-ui/src/Popper/Popper.d.ts
index 4849735ba..2cce5e9cd 100644
--- a/packages/material-ui/src/Popper/Popper.d.ts
+++ b/packages/material-ui/src/Popper/Popper.d.ts
@@ -17,9 +17,11 @@ export type PopperPlacementType =
   | 'top-start'
   | 'top';

+export type AnchorElement = null | ReferenceObject | (() => ReferenceObject)
+
 export interface PopperProps extends React.HTMLAttributes<HTMLDivElement> {
   transition?: boolean;
-  anchorEl?: null | ReferenceObject | (() => ReferenceObject);
+  anchorEl?: AnchorElement;
   children:
     | React.ReactNode
     | ((props: {
```